app/controllers/friendships_controller.rb

```diff
@@ -18,6 +18,14 @@ def destroy
     redirect_to root_path
   end
 
+  # def remove_friendship
+  #   @friendship = Friendship.find_by user_id: current_user.id, friend_id: params[:id]
+  #   @friendship_reverse = Friendship.find_by friend_id: current_user.id, user_id: params[:id]
+  #   @friendship&.destroy
+  #   @friendship_reverse&.destroy
+  #   redirect_to users_path
+  # end
+
```

Comment:
- [ ] Kindly remove commented code, this will make your controller look more clean and professional.